software-setup/slack-procedures.md

```diff
@@ -108,7 +108,7 @@ First, use the new message button in the top right side corner of the Slack inte
 
 <img src = "screenshots/slack-compose-new-message.png" width="300">
 
-You are then able to search for the Data Lab instructors – `Chante Bethell`, `Ally Hawkins`, `Jen O'Malley`, `Josh Shapiro`, or `Stephanie Spielman` – and compose your message.
+You are then able to search for the Data Lab instructors and compose your message.
```

Comment:
Hmmm... maybe we want to make this a list in config and fill these names in automatically? Do we name ourselves elsewhere that we would reuse this?

Reply:
This is the only place according to `grep`, but I'll make a note of this over in #106 in case